### PR TITLE
Fix bug in wildcard - flobject.py

### PR DESCRIPTION
**Pull request by prmukherj**
```
>>> [x for x in solver.setup.boundary_conditions.velocity_inlet]          
['inlet2', 'inlet1']
>>> [x for x in solver.setup.boundary_conditions.velocity_inlet['\*let*']]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'WildcardPath' object is not iterable
```

The above behavior is fixed.

Similar changes have been applied on the PyConsole side as well.

### CODE REVIEW

**Review by hpohekar — APPROVED** [review]

File: src/ansys/fluent/core/solver/flobject.py [hunk 1]
```diff
@@ -572,13 +573,25 @@ def __getattr__(self, name: str):
...
         raise KeyError(
             allowed_name_error_message(
                 "Settings objects", name, self.get_object_names()
             )
         )
 
+    def items(self):
+        """Items."""
+        for key, value in self._parent.items():
+            if fnmatch.fnmatch(key, self._path.split("/")[-1]):
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/solver/flobject.py:587]:
```suggestion
            if fnmatch.fnmatch(key, self._path.rsplit(sep="/", maxsplit=1)[-1]):
```

Comment (seanpearsonuk) [reply src/ansys/fluent/core/solver/flobject.py]: (faster)

File: src/ansys/fluent/core/solver/flobject.py [hunk 2]
```diff
@@ -572,13 +573,25 @@ def __getattr__(self, name: str):
...
+                yield key, value
+
+    def __iter__(self):
+        for item in self._parent:
+            if fnmatch.fnmatch(item, self._path.split("/")[-1]):
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/solver/flobject.py:592]: similar change as above

File: src/ansys/fluent/core/solver/flobject.py [hunk 3]
```diff
@@ -572,13 +573,25 @@ def __getattr__(self, name: str):
...
 
+    def items(self):
+        """Items."""
+        for key, value in self._parent.items():
+            if fnmatch.fnmatch(key, self._path.rsplit(sep="/", maxsplit=1)[-1]):
+                yield key, value
+
+    def __iter__(self):
+        for item in self._parent:
+            if fnmatch.fnmatch(item, self._path.split("/")[-1]):
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/solver/flobject.py:592]:
```suggestion
            if fnmatch.fnmatch(item, self._path.rsplit(sep="/", maxsplit=1)[-1]):
```

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**